docs/src/index.md

```diff
@@ -19,3 +18,8 @@ Localizers.jl                               | Covariance localization kernels
 
 `EnsembleKalmanProcesses.jl` is being developed by the [Climate Modeling
 Alliance](https://clima.caltech.edu). The main developers are Oliver R. A. Dunbar and Ignacio Lopez-Gomez.
+
+## [Experimental]
+
+Here we list any additional usable, but experimental, tools
+- Sparsity-inducing Ensemble Kalman Inversion (SEKI) - Additionally adds approximate ``L^0`` and ``L^1`` penalization to the EKI ([Schneider, Stuart, Wu, 2020](https://doi.org/10.48550/arXiv.2007.06175)). 
```

Comment:
I think it may be better to move this to the end of the ensemble Kalman inversion section, what are your thoughts on that? Sparse EKI is a modification of EKI after all.

Reply:
After discussions we will keep this in the index as it was